charts/prometheus-postgres-exporter/templates/deployment.yaml

```diff
@@ -85,19 +83,19 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           ports:
-            - name: http
+            - name: {{ .Values.service.name }}
               containerPort: {{ .Values.service.targetPort }}
               protocol: TCP
-          livenessProbe:
-            exec:
-              command:
-              - /healthcheck
-              - "{{ .Values.service.targetPort }}"
-          readinessProbe:
-            exec:
-              command:
-              - /healthcheck
-              - "{{ .Values.service.targetPort }}"
+          # livenessProbe:
```

Comment:
i took these out, i will look into if it has health checks and add them back (or if not remove the commented out section).

Reply:
Easy fix for the probes should be something like this
```
    readinessProbe:
      tcpSocket:
        port: {{ .Values.service.targetPort }}
      initialDelaySeconds: 5
      periodSeconds: 10
    livenessProbe:
      tcpSocket:
        port: {{ .Values.service.targetPort }}
      initialDelaySeconds: 15
      periodSeconds: 20
```